Cosmetic file changes

cah-hbaum opened the pull request:
This PR tries to update some files in order to provide some cosmetic and logical file changes, like:
- grammar
- wording
- correct linking
- abbreviations
- general cosmetic changes

I already tried to introduce them in earlier updates, but they were rejected due to mix of concerns, so I created this separate PR for them.

mbuechse reviewed (commented):
I think I agree with 50 % of this. It's great that some obvious typos are fixed, also some shady syntax is fixed (such as the table headings). I really don't like any changes that are up to taste, such as reformatting whole tables, reflowing paragraphs, changing the meaning of things (from hyperthreads to threads, for example). Finally, I don't like the sheer extent of this PR. It does all these things for all the files all at once. This is still complicated to review and discuss, and I'm not even sure there's demand.

cah-hbaum commented:
Removed the changes to tables and paragraph reflows.
Didn't find the threads thing you mentioned, but for good measure, I reverted these things.

At last, I won't do this in smaller batches, since I also don't have the time to split this up, but I'm still gonna keep it, since so many mistakes in these documents seems kind of bad for a project of this size.

mbuechse commented on Standards/scs-0400-v1-status-page-create-decision.md:
```diff
@@ -153,7 +153,7 @@ With those requirements in mind the projects that initially were found, were eva
 | different output format on notification | ❌ | ❌ | ✅ | ✅ | ✅ | ❌ | ✅ |
 | external hosting | ✅ | ✅ | ✅ | ✅ | ✅ | ❌ looks like you are limited to github | ✅ |
-| project healthy | ❌ last commit 17 months | ❌ last commit 3 years | ❌ last commit 5 months | ✅ last commit 2 months | ✅ recent activities | ✅ recent activities | ❌ archived and abondend by the owner |
+| project healthy | ❌ last commit 17 months | ❌ last commit 3 years | ❌ last commit 5 months | ✅ last commit 2 months | ✅ recent activities | ✅ recent activities | ❌ archived and abndend by the owner |
```
```suggestion
| project healthy | ❌ last commit 17 months | ❌ last commit 3 years | ❌ last commit 5 months | ✅ last commit 2 months | ✅ recent activities | ✅ recent activities | ❌ archived and abandoned by the owner |
```

cah-hbaum replied:
Updated in https://github.com/SovereignCloudStack/standards/pull/644/commits/50ab7b2c3b9d9274b29b536e69cda56c6c5c0a9b

mbuechse commented on Standards/scs-0402-v1-status-page-openapi-spec-decision.md:
```diff
@@ -131,21 +131,21 @@ This means:
 Payloads to POST requests SHALL NOT include ID or `Incremental` typed fields, it lies in the responsibility of the API server to assign IDs and `Incremental`s to objects.
 
-### Return of `PATCH` requestes
+### Return of `PATCH` requisites
```
```suggestion
### Return of `PATCH` requests
```

cah-hbaum replied:
Updated in https://github.com/SovereignCloudStack/standards/pull/644/commits/50ab7b2c3b9d9274b29b536e69cda56c6c5c0a9b

mbuechse reviewed (commented):
Yes, very good, thanks very much for this! Please do check my comments.

cah-hbaum commented on Standards/scs-0213-v1-k8s-nodes-anti-affinity.md:
```diff
@@ -8,7 +8,7 @@ track: KaaS
 
 A Kubernetes instance is provided as a cluster, which consists of a set of worker machines,
-so called nodes. A cluster is composed of a control plane and at least one worker node.
+so-called nodes. A cluster is composed of a control plane and at least one worker node.
```
Updated in https://github.com/SovereignCloudStack/standards/pull/644/commits/50ab7b2c3b9d9274b29b536e69cda56c6c5c0a9b

cah-hbaum commented on Standards/scs-0115-v1-default-rules-for-security-groups.md:
```diff
@@ -18,7 +18,7 @@ Security Group (abbr. SG)
 
 Security Group Rule (abbr. SG Rule)
-  A single ip table rule, that is part of a SG.
+  A single ip table rule, that is part of an SG.
```
Replaced SG with Security Group.
Updated in https://github.com/SovereignCloudStack/standards/pull/644/commits/50ab7b2c3b9d9274b29b536e69cda56c6c5c0a9b

cah-hbaum commented on Standards/scs-0114-v1-volume-type-standard.md:
```diff
@@ -93,7 +93,7 @@ openstack volume type show LUKS
 ### Replication
 
-Replication states whether or not there are multiple replicas of a volume. Thus, it answers the question whether the data could survive a node outage. Unfortunately there are two ways replication can be achieved:
+Replication states whether or not there are multiple replicas of a volume. Thus, it answers whether the data could survive a node outage. Unfortunately there are two ways replication can be achieved:
```
Updated in https://github.com/SovereignCloudStack/standards/pull/644/commits/50ab7b2c3b9d9274b29b536e69cda56c6c5c0a9b
